app/models/order.rb

```diff
@@ -0,0 +1,2 @@
+class Order < ApplicationRecord
+end
```

Comment:
Kindly associate the model as well by adding the key_word has_many and belongs_to...